### PR TITLE
Remove the experimental tag from Jolokia autodiscover provider

### PR DESCRIPTION
**Pull request by jsoriano**
I'd say we want to keep Jolokia autodiscover as is, so remove the experimental tag for 7.0.

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -100,6 +100,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...master[Check the HEAD d
 - Add cache.ttl to add_host_metadata. {pull}9359[9359]
 - Add support for index lifecycle management (beta). {pull}7963[7963]
 - Always include Pod UID as part of Pod metadata. {pull]9517[9517]
+- Remove experimental tag from Jolokia autodiscover provider. {pull}9706[9706]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]:
```suggestion
 - Release Jolokia autodiscover as GA. {pull}9706[9706]
```

**Comment by jsoriano** [comment]
jenkins, test this again please

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -200,6 +200,12 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...v7.0.0-alpha2[Check the
 - Unify dashboard exporter tools. {pull}9097[9097]
 - Add cache.ttl to add_host_metadata. {pull}9359[9359]
 - Add support for index lifecycle management (beta). {pull}7963[7963]
+- Always include Pod UID as part of Pod metadata. {pull]9517[9517]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: Rebase issue?

Comment (jsoriano) [reply CHANGELOG.asciidoc]: Yes, there was something wrong there, fixed.

**Comment by jsoriano** [comment]
@ruflin I was thinking on doing this for 7.0, but I could backport it for 6.7, as you prefer

**Comment by ruflin** [comment]
I assume there are no code changes in this for 6.6 / 6.7 or master? If yes, let's backport. I'm even thinking about updating 6.6?

**Comment by jsoriano** [comment]
Ok, I will backport it till 6.6 :slightly_smiling_face:

**Pull request merged**